content/posts/2022-09-14-introducing-jaiminho/index.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Introducing Jaiminho, an implementation of the Transactional Outbox Pattern for Django
+author: Rafael Zanetti
+twitter: LoadsmartEng
```

Comment:
Do you have a Twitter account @rfzanetti ?

Reply:
I do, but it has too much dumb stuff and not to much tech stuff haha I could also create an account just to link it here, what do you suggest?